integration/yarn/src/main/java/alluxio/yarn/Client.java

```diff
@@ -394,13 +394,13 @@ private void setupContainerLaunchContext() throws IOException, YarnException {
 
   private void setupAppMasterEnv(Map<String, String> appMasterEnv) throws IOException {
     String classpath = ApplicationConstants.Environment.CLASSPATH.name();
+    Apps.addToEnvironment(appMasterEnv, classpath, PathUtils.concatPath(Environment.PWD.$(), "*"),
+        ApplicationConstants.CLASS_PATH_SEPARATOR);
```

Comment:
Could you add a comment to make sure we know the ordering of this matters?

Reply:
jinx